functions/exercise4/scriptSpec.js

```diff
@@ -0,0 +1,17 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals validateAsPhraseConsistsOfLettersOnly*/
+
+describe("Check if phrase consists of letters only.", function () {
+  it("white space ' ' is an exception", function () {
+    expect(function () { validateAsPhraseConsistsOfLettersOnly(" "); }).toThrow(new Error("The parameter should be consist only of letters."));
+  });
+  it("number 1 is an exception", function () {
+    expect(function () { validateAsPhraseConsistsOfLettersOnly(1); }).toThrow(new Error("The parameter should be consist only of letters"));
+  });
+  it("number '1' is an exception", function () {
+    expect(function () { validateAsPhraseConsistsOfLettersOnly("1"); }).toThrow(new Error("The parameter should be consist only of letters"));
```

Comment:
Please add . at the end of the error - you used . in line 6 and it would be good to have unified message.